python-mip/mip_helper/mip_helper/parameters.py

```diff
@@ -30,6 +30,10 @@ def get_param(param_name, param_type, default_value):
     return param_type(default_value)
 
 
+# TODO: should we use name `get_parameter` or `get_param`?
+get_parameter = get_param
```

Comment:
get_parameter is better, I'm against shortcuts in APIs. And get_boolean_param will need to be renamed as well

Reply:
👍 I'll change it then and `get_boolean_param` as well and make it backward compatible.